src/hci/gnomex/controller/GetWorkItemList.java

```diff
@@ -635,7 +628,7 @@ private void fillSeqPrepQC(Element n, Object[] row, String codeRequestCategory)
     n.setAttribute("idLibPrepQCProtocol", row[WorkItemFilter.ID_LIB_PREP_QC_PROTOCOL] == null ? "" : ((Integer)row[WorkItemFilter.ID_LIB_PREP_QC_PROTOCOL]).toString());
   }
 
-  private void fillSeqPrep(Element n, Object[] row, String codeRequestCategory, HashMap<String, Integer> seqLibProtocolMap) {
+  private void fillSeqPrep(Element n, Object[] row, String codeRequestCategory, HashMap<String, Integer> seqLibProtocolMap, Session sess) {
```

Comment:
It looks like you added a Session object to the method signature but didn't end up needing it.